test/data/README.md

```diff
@@ -26,7 +30,7 @@ Redistributing the dataset "breast-cancer.txt" with attribution:
 
 ### MNIST
 
-> MNIST data originally from [NIST](https://www.nist.gov) and modified by Chris Burges, Corinna Cortes, and Yann LeCun. http://yann.lecun.com/exdb/mnist/
+> MNIST data originally from [NIST](https://www.nist.gov) and modified by Chris Burges, Corinna Cortes, and Yann LeCun.e http://yann.lecun.com/exdb/mnist/
```

Comment:
❔  Was this an intentional change to the name?

Reply:
No :) , fixing.